rwatch/ui/layer/menu_layer.c

```diff
@@ -367,7 +367,9 @@ static void menu_layer_draw_cell(GContext *context, const MenuLayer *menu_layer,
     if (menu_layer->callbacks.draw_background)
         menu_layer->callbacks.draw_background(context, layer, highlighted, menu_layer->context);
     else
+#ifdef PBL_RECT
```

Comment:
Using a preprocessor if here causes the if statement in line 375 now is an else-if statement, which in turn causes funny bugs. For example if you are on chalk and decide to supply a `draw_background` callback of your own, the content is never drawn.